doc: known_issues: Add NCSDK-22953 (#11955)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/known_issues.rst

  **Workaround:** Use BlueZ in version 5.56 or higher.

.. rst-class:: v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0

> NCSDK-22953: An HID output report interrupts scanning in the :ref:`nrf_desktop_ble_scan` even if there are no peripherals connected

Reviewer: ```suggestion
NCSDK-22953: A HID output report interrupts scanning in the :ref:`nrf_desktop_ble_scan` even if there are no peripherals connected
```
let's stick to "hid" pronunciation for now, still in the process of checking this case of article:)

>   Connecting a HID dongle over USB to a HID host may lead to submitting a HID output report and stopping Bluetooth LE scanning even if there are no peripherals connected through the dongle.

Reviewer: ```suggestion
  Connecting a HID dongle over USB to a HID host might lead to submitting a HID output report and stopping Bluetooth LE scanning even if there are no peripherals connected through the dongle.
```

>   Scanning stop in that case delays establishing connection with peripherals.

Reviewer: ```suggestion
  In this case, scanning stop delays establishing a connection with peripherals.
```
